src/embedded_flash/store/mod.rs

```diff
@@ -728,6 +770,22 @@ impl<C: StoreConfig> Store<BufferStorage, C> {
     pub fn set_erase_count(&mut self, page: usize, erase_count: usize) {
         self.initialize_page(page, erase_count);
     }
+
+    /// Checks whether all deleted sensitive entries have been wiped.
+    pub fn check_wiped(&self) {
+        for (_, entry) in Iter::new(self) {
+            if !self.format.is_present(entry)
+                || !self.format.is_deleted(entry)
+                || self.format.is_internal(entry)
+                || !self.format.is_sensitive(entry)
+            {
+                continue;
+            }
+            let gap = self.format.entry_gap(entry);
+            let data = gap.slice(entry);
+            assert!(data.iter().all(|&byte| byte == 0x00));
```

Comment:
Do we want to panic here?
Because the device will potentially be bricked at that point, right: at each boot it may try to wipe data and panic.

Reply:
This function is only used in unit tests on the desktop.

Reply:
I changed the function to return a `bool` such that the assert only appears in tests and fuzzing.